Formula/php.rb

```diff
@@ -20,15 +20,18 @@ class Php < Formula
   depends_on "curl-openssl"
   depends_on "freetds"
   depends_on "freetype"
+  depends_on "gd"
   depends_on "gettext"
   depends_on "glib"
   depends_on "gmp"
   depends_on "icu4c"
   depends_on "jpeg"
+  depends_on "libffi"
   depends_on "libpng"
   depends_on "libpq"
   depends_on "libsodium"
   depends_on "libzip"
+  depends_on "oniguruma"
```

Comment:
This should be bundled by PHP

Reply:
Build fails without:

> checking for oniguruma... no
configure: error: Package requirements (oniguruma) were not met:
>
> No package 'oniguruma' found
>
> Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
>
> Alternatively, you may set the environment variables ONIG_CFLAGS
and ONIG_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

Given that, I'd keep it in.

Reply:
Indeed Oniguruma is no longer bundled.